examples/jsm/loaders/LUTImageLoader.js

```diff
@@ -0,0 +1,160 @@
+// http://download.autodesk.com/us/systemdocs/help/2011/lustre/index.html?url=./files/WSc4e151a45a3b785a24c3d9a411df9298473-7ffd.htm,topicNumber=d0e9492
+// https://community.foundry.com/discuss/topic/103636/format-spec-for-3dl?mode=Post&postID=895258
+import {
+	Loader,
+	TextureLoader,
+	DataTexture,
+	Data3DTexture,
+	RGBAFormat,
+	UnsignedByteType,
+	ClampToEdgeWrapping,
+	LinearFilter,
+} from 'three';
+
+export class LUTImageLoader extends Loader {
+
+	constructor( flipVertical = true ) {
+
+		super();
+
+		this.flip = flipVertical;
```

Comment:
Can you explain the rationale behind "flip"? Is it necessary? It looks like it just flips y regardless of whether the png is wider or taller - which changes the color axis that gets flipped (green if horizontal, blue if vertical). Unless there's a good reason or common case where it's necessary.

Reply:
To convert the image into a LUT it is necessary to flip in the y in most cases. I made this the default but changable if a user has LUT images that have blocks with green at the bottom. All the images I've seen have blocks in a horizontal direction. I assumed vertical would still have the blocks orientated as horizontal but stacked vertically rather than horizontally. So flip y is still the default.

Reply:
> I assumed vertical would still have the blocks orientated as horizontal but stacked vertically rather than horizontally. So flip y is still the default.

I'm not sure what this means exactly - if we have a vertical strip the blue axis would be increasing from top to bottom rather than left to right. And therefore flipping Y would reverse blue, as well. If vertical LUTs don't seem to be available or used anywhere else I'm wondering if it's worth supporting at all to avoid promoting new bespoke formats and we have no test data.